openstack/autoscaling/v1/configurations/results.go

```diff
@@ -24,9 +24,11 @@ type GetResult struct {
 }
 
 func (r GetResult) Extract() (Configuration, error) {
-	var a Configuration
-	err := r.Result.ExtractIntoStructPtr(&a, "scaling_configuration")
-	return a, err
+	var s struct {
+		ScalingConfiguration Configuration `json:"scaling_configuration"`
+	}
+	err := r.ExtractInto(&s)
```

Comment:
You can use `ExtractIntoStructPtr(&c, "scaling_configuration")` instead of introducing new structure

Reply:
done